### DISCUSSION TITLE
some setting not translatable ?

### DISCUSSION THREAD

**Discussion by Moosh-be** [bluecaret/carettab#359] [Help and Support]
![image](https://github.com/bluecaret/carettab/assets/388085/c4347789-9e71-4417-b161-a84a191732b7)

When I switch to french, some part of settings still in english. 
And I don't found it in fr.json

`  <div class="label">
        <label id="containerAlignLabel">{{
          props.containerTitle ? props.containerTitle : 'Container alignment and size'
        }}</label>
        <div class="desc">
          {{
            props.noContainerAlignment
              ? 'Sets the size of the widget container'
              : 'Sets the size of the widget container and alignment of content within the container'
          }}
        </div>
      </div>`

https://github.com/bluecaret/carettab/blob/edeb539cb54c9500207b1f3b2ee0ec78b6db5139/src/components/forms/SizeAndPositionField.vue#L189

**Comment by bluecaret** [comment]
Thank you for the report, does look like I forgot to convert that string to be translated. I'll get that fixed for the next update. If you are able to provide what a proper translation for that in French would be here that would be helpful :)

**Reply by Moosh-be** [reply]
![image](https://github.com/bluecaret/carettab/assets/388085/5d6efafa-521d-490e-8c5c-ad514b16b2aa)

1 - Alignement et taille des conteneurs

2 - Définit la taille du conteneur du widget et l’alignement du contenu dans le conteneur

3 - Largeur

4 - Hauteur

**Comment by bluecaret** [comment]
Thank you, this will be in the next update.

**Reply by bluecaret** [reply]
This update has just been published and will be out in a few days.
